lexicon/conversion.py

```diff
@@ -140,6 +140,118 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from space or tab separated files.
```

Comment:
Maybe describe the input format a little bit more, something like:
Each line contains
< WORD > < PHONEME1 > < PHONEME2 > ...
separated by tabs or spaces

Reply:
Also the code below assumes that the entries are sorted alphabetically.